my-dsl.md

```diff
@@ -2,37 +2,41 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
++ I chose the SumoLogic query language
++ https://service.us2.sumologic.com/help/Default.htm#Search_Example_Cheat_Sheets.htm
+
 
 # Domain
 _Describe the language's domain in five words._
 
-
-# Computational model
-_We don't yet have a great definition of the term "computational model". 
-For now, try to come up with the clearest, most concise explanation of 
-what happens when a program in your DSL runs._
-
++ Searching / Querying application log data
 
```

Comment:
I think you accidentally deleted the question about the computational model of your language.